packages/react-utilities/src/hooks/useTimeout.test.ts

```diff
@@ -2,20 +2,31 @@ import { renderHook } from '@testing-library/react-hooks';
 import { useTimeout } from './useTimeout';
 
 describe('useTimeout', () => {
-  jest.useFakeTimers();
+  beforeEach(() => {
+    jest.useFakeTimers();
+  });
+
+  afterEach(() => {
+    jest.useRealTimers();
+  });
 
   it('calls the underlying setTimeout/clearTimeout API', () => {
     const [setTestTimeout, clearTestTimeout] = renderHook(() => useTimeout()).result.current;
     const callback = jest.fn();
 
+    // React internals may also call setTimeout
```

Comment:
this complexity of test demonstrates that this is probably a bad/not needed test. 
I'd remove it. 

From implementation perspective/usage perspective it is not important what underlying mechanism are triggering the delays.

Reply:
Makes sense, I'll remove it